catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/actionbar/utils/ActionModeWrapper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.ui.actionbar.utils;
+
+import org.catrobat.catroid.R;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public final class ActionModeWrapper {
+	private ActionModeWrapper() { }
+
+	public static ActionModeWrapper onActionMode() {
+		return new ActionModeWrapper();
+	}
+
+	public ActionModeWrapper checkTitleMatches(String text) {
+		onView(withId(R.id.action_bar_title)).check(matches(withText(text)));
+		return this;
+	}
+
+	public ActionModeWrapper performGoBack() {
+		onView(withId(R.id.action_mode_close_button)).perform(click());
+		return this;
```

Comment:
here i wouldnt return anything since actionbar should be gone and nothing can be done anymore with this object.